### ISSUE TITLE
Macroparasite models

### ISSUE THREAD

**Issue by epichef** [epirecipes/shopping-list#25]

**Comment by Federica-Giardina** [comment]
I'd like to implement a basic macroparasite model (May and Anderson)

**Comment by ctedijanto** [comment]
I would like to work on the schistosomiasis model here (Woolhouse): https://doi.org/10.4269/ajtmh.1996.55.144

**Comment by sdwfrost** [comment]
Hi @federigia - in R? We have May and Anderson in the Enigma room.

**Comment by Federica-Giardina** [comment]
Yes! I did it in R but interested in trying a Julia implementation too

**Comment by sdwfrost** [comment]
Added in [https://github.com/epirecipes/epicookbook/commit/479e20461d0de77423c4a3df41c9e22723adf87f](https://github.com/epirecipes/epicookbook/commit/479e20461d0de77423c4a3df41c9e22723adf87f) but keeping this issue open as we need more macroparasite models.
